elabapy/Manager.py

```diff
@@ -24,23 +24,23 @@ def create_template(self):
         """
         return self.send_req("templates", verb='POST')
 
-    def get_all_experiments(self):
+    def get_all_experiments(self, params={'limit':15,'offset':0}):
         """
             This function returns a list of all experiments.
         """
-        return self.send_req("experiments/")
+        return self.send_req("experiments/", params, param_name='params')
```

Comment:
param_name has to be 'params' to be in accordance to the requests [doku](https://2.python-requests.org/en/master/api/#requests.get)